src/app/components/main-action/main-action.component.html

```diff
@@ -0,0 +1,13 @@
+<ng-container *ngIf="(mainAction$ | async) as action">
```

Comment:
Not sure what approach do we have, checking if the element is available within the element, or in place where it is used e.g.  
```
<app-main-action *ngIf="...">
```

Reply:
LGTM